deploy/grafana-dashboard/utils/color_mapping.py

```diff
@@ -19,3 +23,13 @@ def color_mapping(name, color):
 
 
 grey_all_mapping = color_mapping("all", "#4c4b5a")
+color_mapping_environments = {}
+colours = ["yellow", "blue", "orange", "red", "purple",
+           "pink", "brown", "black", "white", "gray",
+           "gold", "super-light-red", "dark-red", "dark-orange", "super-light-yellow",
+           "super-light-green", "dark-green", "dark-blue", "super-light-purple", "super-light-blue"]
+
+with open("environments.json") as f:
+    environments = json.load(f)
```

Comment:
I would propose sorting the list, so that colors are more independent of the sorting in the JSON (and stable)

```suggestion
    environments = json.load(f)
    environments.sort()
```